src/controllers/product-controller.js

```diff
@@ -1,4 +1,79 @@
-// eslint-disable-next-line no-unused-vars
+const config = require('config');
+const FileType = require('file-type');
+const Promise = require('bluebird');
+
 const { ProductModel } = require('../models');
+const S3Lib = require('../libs/s3-lib');
+const NotFoundError = require('../errors/not-found-error');
+const ForbiddenError = require('../errors/forbidden-error');
+
+
+async function uploadImages(req, res, next) {
+  try {
+    const { accessKeyId, secretAccessKey, bucketName } = config.get('aws');
+
+    const { productId } = req.params;
+    const { _id } = req.userData;
+    const product = await ProductModel.findOne({ _id: productId }).select('userId');
+    let distFileKeys = [];
+    let urls = [];
+
+    // Check product exists in db
```

Comment:
variable name already says what we do, there is no need to comment simple code blocks